ng2-material/components/data_table/data_table_tr.ts

```diff
@@ -64,15 +64,17 @@ export class MdDataTableTr implements AfterContentInit {
   }
 
   ngAfterContentInit() {
-    let element = this._element.nativeElement;
-    this.isInHeader = element.parentElement.localName === 'thead';
-    this._initListeners();
+    if (null !== this.table && undefined !== this.table) {
```

Comment:
You should import and use the isPresent helper that angular2 exports instead of two explicit checks.

```
if (isPresent(this.table)) {
  ...
}
```